src/nft/components/explore/CarouselCard.tsx

```diff
@@ -113,6 +108,22 @@ const CollectionImage = styled.img`
   border-radius: 100px;
 `
 
+const LoadingCardHeaderContainer = styled.div`
```

Comment:
Could you add some context for what this loading change is for?

Reply:
the old loading component used the same wrapper as the non-loading component, so if we use `link` (which we should) it demands a `to` prop which we don't want it to have. I also re-organized the styled-components definitions to group the loading<name> components.